hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -145,11 +145,15 @@ public static DFSPropertiesConfiguration readConfig(org.apache.hadoop.conf.Confi
    * Returns the payload config with given configuration.
    */
   public static HoodiePayloadConfig getPayloadConfig(Configuration conf) {
-    return HoodiePayloadConfig.newBuilder()
+    HoodiePayloadConfig.Builder builder = HoodiePayloadConfig.newBuilder()
         .withPayloadClass(conf.getString(FlinkOptions.PAYLOAD_CLASS_NAME))
-        .withPayloadOrderingField(conf.getString(FlinkOptions.PRECOMBINE_FIELD))
-        .withPayloadEventTimeField(conf.getString(FlinkOptions.PRECOMBINE_FIELD))
-        .build();
+        .withPayloadOrderingField(conf.getString(FlinkOptions.PRECOMBINE_FIELD));
```

Comment:
We can just fix the `conf.getString(FlinkOptions.PRECOMBINE_FIELD)` with a fallback value, can you also write a basic IT in `ITTestHoodieDataSource` ?